### DISCUSSION TITLE
backend relation list

### DISCUSSION THREAD

**Discussion by as7-ali-soltani** [wintercms/winter#1088] [Q&A] [answered]
Hi i use this table that use relation with pivot data
![Untitled Diagram (1)](https://github.com/wintercms/winter/assets/44004362/da5613bf-82bf-4056-9ae0-e53beb44d023)
Belongs to many (with Pivot Data):Clicking Add will display a **selection list** (manage.list).
How to use in **selection list** 2 columns with switch type that first switch option load from property model and second switch is depend on first and load from attr model and add one item at a time

**Comment by mjauvin** [comment]
Can you give the name of the model that uses the belongsToMany relation and to which model ?

**Reply by as7-ali-soltani** [reply]
```
class Variant extends Model
{
    public $table = 'as7_variant2_variant';

    public $belongsToMany = [
        'attribute' => [
           Attr::class,
           'table' => 'as7_variant2_attr_variant'
          ]
    ];
}
```

**Comment by mjauvin** [comment, marked as answer]
I think you want to have the Relation Controller two level deep which is currently not supported.

There is a tutorial on how to implement this here:

https://octobercms.com/support/article/ob-21
